.github/workflows/build-pkg.yml

```diff
@@ -0,0 +1,43 @@
+name: Build Homebrew package
+on:
+  push:
+    paths:
+      - .github/workflows/build-pkg.yml
+      - package/scripts
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    runs-on: macos-12
+    env:
+      IDENTIFIER: sh.brew.Homebrew
+      TMP_PATH: /tmp/brew
+      MIN_OS: '11.0'
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: brew
+          fetch-depth: 0
+      - name: Version name
+        id: print-version
+        run: |
+          echo "version=$(git -C brew describe --tags --always)" > $GITHUB_OUTPUT
+      - name: Build package
+        run: |
+         pkgbuild --root brew \
+                  --scripts brew/package/scripts \
+                  --install-location "$TMP_PATH" \
+                  --identifier "$IDENTIFIER" \
+                  --min-os-version "$MIN_OS" \
+                  --filter .DS_Store \
+                  --version ${{ steps.print-version.outputs.version }} \
+                  Homebrew-${{ steps.print-version.outputs.version }}.pkg
+      - uses: actions/upload-artifact@v3
+        with:
+          name: Homebrew ${{ steps.print-version.outputs.version }}
+          path: Homebrew-${{ steps.print-version.outputs.version }}.pkg
```

Comment:
Do we need to sign this?

Reply:
That would be my next step, but we don't need to

Reply:
Ok sounds good. Wasn't sure what the restrictions were on Apple Silicon - I'm guessing it could easily already be ad-hoc signed anyway.